lib/theme-options/site-options.php

```diff
@@ -24,7 +24,7 @@
       'name'    => __( 'Evento Actual/Siguente', 'IGV' ),
       'id'      => $prefix . 'radio_evento_current',
       'type'    => 'select',
-      'show_option_none' => false,
+      'show_option_none' => true,
```

Comment:
Well you can merge this if you want, but I haven't actually tested it as the select wouldn't work. See what happens to @mikeyrayvon first I think